packages/protocol-kit/package.json

```
@@ -81,7 +81,7 @@
     "@ethersproject/address": "^5.7.0",
     "@ethersproject/bignumber": "^5.7.0",
     "@ethersproject/solidity": "^5.7.0",
-    "@safe-global/safe-deployments": "^1.22.0",
+    "@safe-global/safe-deployments": "^1.23.0",
```

`v1.24.0` is already [available](https://github.com/safe-global/safe-deployments/releases) so let's use that one
```suggestion
    "@safe-global/safe-deployments": "^1.24.0",
```

---

yes make sense, will update soon